lib/discourse_automation/scriptable.rb

```diff
@@ -228,8 +228,9 @@ def self.send_pm(
             return
           end
 
-          if (pm[:target_usernames] || []).empty? && (pm[:target_group_names] || []).empty?
-            Rails.logger.warn "[discourse-automation] Did not send PM - no target usernames or groups"
+          if (pm[:target_usernames] || []).empty? && (pm[:target_group_names] || []).empty? &&
```

Comment:
All of these can be changed to be slightly more readable with `Array.wrap(pm[:target_usernames]).empty?`

Reply:
Nice, I should probably use Array.wrap more, feels cleaner